MrtCore/winui_desktop_packaged_app/winui_desktop_packaged_app.sln

```diff
@@ -47,6 +52,18 @@ Global
 		{AF3AA2D2-7150-4996-8C38-0AB44604C3D0}.Release|x64.Build.0 = Release|x64
 		{AF3AA2D2-7150-4996-8C38-0AB44604C3D0}.Release|x86.ActiveCfg = Release|x86
 		{AF3AA2D2-7150-4996-8C38-0AB44604C3D0}.Release|x86.Build.0 = Release|x86
+		{E52B115E-E54D-4BF6-8174-4FFF5E8860BE}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
+		{E52B115E-E54D-4BF6-8174-4FFF5E8860BE}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{E52B115E-E54D-4BF6-8174-4FFF5E8860BE}.Debug|x64.ActiveCfg = Debug|Any CPU
+		{E52B115E-E54D-4BF6-8174-4FFF5E8860BE}.Debug|x64.Build.0 = Debug|Any CPU

```

Comment:
>Any CPU [](http://example.com/codeflow?start=67&length=7)

should these be x64 or anycpu?

Reply:
Default for everything in the solution is Any CPU right now. That results in x86 on my dev box, which seems fine to me. If you want me to change the default to x64, I can.

Reply:
fine, as long as it has no build warning and runs.